src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -91,4 +94,52 @@ std::string HttpInfo::toJson()
   return jh.str();
 }
 
+
+
+/* ****************************************************************************
+*
+* HttpInfo::fill -
+*/
```

Comment:
In my opiniong the decission should be done by the author. In some case, having all the time `blabla::` makes the code harder to read and follow. In some cases, it doesn't. So I'm not sure that could be defined as a general rule for .cpp files.

Let's be pragmatic: if you read `BSONObj` in our code there is no doubt where that `BSONObj` came from. There isn't any risk of confussion between the `mongo` namespace or any other.

However, what should be a general rule (in MUST category) is that `using namespaces` cannot be used in .h files. That could cause conflicts (we have had some of those in the pasts). In fact, maybe we already have this one?